src/main/java/org/opensrp/web/Constants.java

```diff
@@ -19,16 +21,16 @@
     String FAILURE = "Failure occurred";
     String DEFAULT_EXCEPTION_HANDLER_MESSAGE = "The server encountered an error processing the request." ;
 
-    public static final String RETURN_COUNT= "return_count";
-    public static final String TOTAL_RECORDS = "total_records";
+    String RETURN_COUNT= "return_count";
+    String TOTAL_RECORDS = "total_records";
 
-    public static final String PAGE_NUMBER = "pageNumber";
+    String PAGE_NUMBER = "pageNumber";
 
-    public static final String PAGE_SIZE = "pageSize";
+    String PAGE_SIZE = "pageSize";
 
-    public static final String ORDER_BY_TYPE = "orderByType";
+    String ORDER_BY_TYPE = "orderByType";
 
-    public static final String ORDER_BY_FIELD_NAME = "orderByFieldName";
+    String ORDER_BY_FIELD_NAME = "orderByFieldName";
 
     String LOCATIONS = "locations";
```

Comment:
Any reason we are changing the access modifies for these fields?

Reply:
I performed code clean up on the code. Those are access modifiers applied in a public interface which is not necessary.